src/TCommandLine.cpp

```diff
@@ -1028,10 +1039,11 @@ void TCommandLine::handleTabCompletion(bool direction)
             filterList.removeAll(tmp);
             filterList.insert(offset, tmp);
             ++offset;
-            if (offset >= filterList.size()) {
+            if (offset >= filterList.size()){
```

Comment:
Bogus loss of whitespace... :laughing: 
```suggestion
            if (offset >= filterList.size()) {
```